Connection/Connection.php

```diff
@@ -82,6 +88,7 @@ public function getMailbox(): Mailbox
                 $this->password,
                 $this->attachmentsDir,
                 $this->serverEncoding,
+                $this->enabled,
```

Comment:
As I can see the 6th argument of the [Mailbox::__construct()](https://github.com/barbushin/php-imap/blob/94107fdd1383285459a7f6c2dd2f39e25a1b8373/src/PhpImap/Mailbox.php#L168) is  $trimImapPath, why are you passing enabled to it?